libfwupdplugin/fu-self-test.c

```diff
@@ -1945,6 +1945,38 @@ fu_strtoull_func(void)
 	g_assert_false(ret);
 }
 
+static void
+fu_strtoll_func(void)
+{
+	gboolean ret;
+	gint64 val = 0;
+	g_autoptr(GError) error = NULL;
+
+	ret = fu_strtoll("123", &val, 123, 200, &error);
+	g_assert_no_error(error);
+	g_assert_true(ret);
+	g_assert_cmpint(val, ==, 123);
+
+	ret = fu_strtoll("-123\n", &val, -123, 200, &error);
+	g_assert_no_error(error);
+	g_assert_true(ret);
+	g_assert_cmpint(val, ==, -123);
+
+	ret = fu_strtoll("-0x123", &val, -0x123, 0, &error);
+	g_assert_no_error(error);
+	g_assert_true(ret);
+	g_assert_cmpint(val, ==, -0x123);
```

Comment:
negative hex number?  I've never seen that.

Reply:
Fixed.